pkg/scheduler/util/utils.go

```diff
@@ -163,3 +164,9 @@ func ClearNominatedNodeName(cs kubernetes.Interface, pods ...*v1.Pod) utilerrors
 	}
 	return utilerrors.NewAggregate(errs)
 }
+
+// IsScalarResourceName validates the resource for Extended, Hugepages, Native and AttachableVolume resources
+func IsScalarResourceName(name v1.ResourceName) bool {
```

Comment:
I'm starting to wonder if we really need this check. Which resources would fail it, other than the ones we already pre-fetch early in `Resource.Add` or `Resource.Max`?